lib/m2ts/caption-stream.js

```diff
@@ -216,6 +216,14 @@ var BASIC_CHARACTER_TRANSLATION = {
   0x7f: 0x2588
 };
 
+var getCharFromCode = function(code) {
+  if(!code) {
```

Comment:
This would also be falsey for code `0` which David thinks might be important one day. I would make this explicity `code !== null`